common/neon/block.py

```diff
@@ -29,6 +33,7 @@ def default(cls) -> Self:
             block_time=None,
             parent_slot=None,
             parent_block_hash=EthBlockHashField.default(),
+            priority_fee_percentiles=[0] * 11,  # Although this path is not used, let's make it future-proof.
```

Comment:
Please add the constant to the class definition instead of magic number:
 ```
 class NeonBlockHdrModel(BaseModel):
    PercentileCnt: Final[int] = 11
    ....
```